libc/test/src/stdbit/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+add_custom_target(libc-stdbit-tests)
+
+add_libc_test(
+  stdc_leading_zeros_uc_test
+  SUITE
+    libc-stdbit-tests
+  SRCS
+    stdc_leading_zeros_uc_test.cpp
+  DEPENDS
+    libc.src.stdbit.stdc_leading_zeros_uc
+)
+
+add_libc_test(
+  stdc_leading_zeros_us_test
+  SUITE
+    libc-stdbit-tests
+  SRCS
+    stdc_leading_zeros_us_test.cpp
+  DEPENDS
+    libc.src.stdbit.stdc_leading_zeros_us
+)
+
+add_libc_test(
+  stdc_leading_zeros_ui_test
+  SUITE
+    libc-stdbit-tests
+  SRCS
+    stdc_leading_zeros_ui_test.cpp
+  DEPENDS
+    libc.src.stdbit.stdc_leading_zeros_ui
+)
+
+add_libc_test(
+  stdc_leading_zeros_ul_test
+  SUITE
+    libc-stdbit-tests
+  SRCS
+    stdc_leading_zeros_ul_test.cpp
+  DEPENDS
+    libc.src.stdbit.stdc_leading_zeros_ul
+)
+
+add_libc_test(
+  stdc_leading_zeros_ull_test
+  SUITE
+    libc-stdbit-tests
+  SRCS
+    stdc_leading_zeros_ull_test.cpp
+  DEPENDS
+    libc.src.stdbit.stdc_leading_zeros_ull
```

Comment:
Can you add `libc.src.__support.CPP.limits` to their lists of dependency?

Reply:
I've removed this dependency in https://github.com/llvm/llvm-project/commit/3386377534329227a20f4924f882e63940f28ea0, but please triple check.